### ISSUE TITLE
Notify a player in the launcher if their account is banned from multiplayer

### ISSUE THREAD

**Issue by TheMCNerd2017** [PolyMC/PolyMC#824]
### Role

Being banned from Minecraft Java multiplayer

### Suggestion

Tell players if their account is banned from multiplayer

### Benefit

So players don't need to launch the latest versions of the game to see if they're banned as well as details of the ban

### This suggestion is unique

- [X] I have searched the issue tracker and did not find an issue describing my suggestion, especially not one that has been rejected.

### You may use the editor below to elaborate further.

EDIT 3: I rewrote the suggestion so it's easier to read and is not a complete mess like it was previously. It's also more detailed.

In 1.19.1, game-wide multiplayer bans that span across all versions of the game are being introduced. If you are banned by Mojang/Microsoft, you will get an in-game notice upon launching 1.19.1 or newer telling you that you're banned as well as giving some extra information, like duration of the ban as well as the reason. I suggest that the launcher includes the ability to tell you if your account is banned for a few reasons.

1. It would allow players to see if they are banned without having to launch the game.
2. People with multiple accounts could see if any of their accounts has gotten banned or not.
3. People that can't or don't play 1.19.1 or newer for any reason can see if they are banned as well as the details of the ban.
4. Bans are authentication-based, and while you will get an authentication error on older versions if you are banned and try to join a server, important details like the reasoning and duration may not be provided in the error. Additionally some people might not know about the error and assume something else is wrong.

As for telling the player that their account is banned, upon adding a Microsoft account the launcher could tell you if the account is banned from multiplayer via an information box containing similar information to the 1.19.1 in-game notice. The same information box would also appear when launching an instance with the selected Microsoft account. A checkbox would also be present in the information box which would allow you to stop the information box from coming up on future launches of the game with that specific account. Removing and re-adding the affected Microsoft account would make the information box come up again, however that could be changed if removing and re-adding accounts to the launcher's account list is a common thing to do.

Additionally, any Microsoft account that is banned from multiplayer would have a special icon appear over the profile icon in the bottom right area of the icon (I'm thinking a red prohibitory sign would do for now) and the Status of the account would show up as "Banned" instead of Online to indicate the account is banned from multiplayer and is essentially an offline/cracked account functionality-wise.

I'll also make a mockup picture of what some of the functionality would look like when I get the chance to do so.

EDIT 4: I made a few image mockups. 

First image is of what a multiplayer banned account would look like on the main page. Currently a red prohibitory symbol since I don't know what symbol would fit. 
![ban_notice1](https://user-images.githubusercontent.com/25375242/174930797-b08c7f41-aa60-4d31-8b3e-9b230ececbac.jpg)

Second image is of the "Banned" status for accounts banned from multiplayer. An offline account is present in the mockup, however the "Banned" status would only be possible for real accounts that could actually play online.
![ban_notice2](https://user-images.githubusercontent.com/25375242/174931090-47b3c345-d6d7-41bc-aecd-926acb1ea679.jpg)

**Comment by DioEgizio** [comment]
The game literally already does this, so it makes no sense to do it

**Comment by TheMCNerd2017** [comment]
> The game literally already does this, so it makes no sense to do it

Yes, however the point of this is so the player doesn't have to launch the game to see if and why they are banned. I mentioned that in the benefit section.

**Comment by copygirl** [comment]
Wouldn't they only have to launch the game once to know? It sounds like the main purpose of this suggestion might be bulk checking of many accounts. Why would a legitimate user need to be notified by their launcher otherwise?

**Comment by TheMCNerd2017** [comment]
> Wouldn't they only have to launch the game once to know? It sounds like the main purpose of this suggestion might be bulk checking of many accounts. Why would a legitimate user need to be notified by their launcher otherwise?

Yeah they would have to launch the game to know, however it can be slow to launch the game, especially if you are launching it just to see a ban message. And no, the purpose of this isn't for cheating (you can't be banned by Mojang/Microsoft for cheating, only for sending chat messages that violate their guidelines) it was merely a convenience feature.

EDIT: I misread your comment and thought you said cheating instead of checking.

**Comment by RaptaG** [comment]
The multiplayer banning is one of **the worst** features Mojang has added to the game

**Comment by TheMCNerd2017** [comment]
> The multiplayer banning is one of **the worst** features Mojang has added to the game

I agree, and it's worse if you only play on older versions since you'd have no idea if you're even banned or not.
EDIT: You actually will have some sort of idea if you're banned on older versions, as you'll get an error when attempting to join any servers that are not in offline-mode.

**Comment by d-513** [comment]
But if a user doesn’t play new versions, they won’t be notified just have errors. Having a ban check in accounts list is very useful

**Comment by DioEgizio** [comment]
But is it possible?

**Comment by d-513** [comment]
> But is it possible?

There’s sussy stuff in the profile response so probably. Can also just decompile mc and see how it’s doing that

**Comment by svin24** [comment]
I hope i am not hijacking this thread too much. What are the devs opinions on adding a "cracked" mode?
(or will we have to make forks/patches for that)

Its just that these new rules are too far reaching(affecting even private servers which microsoft does not even own).

**Comment by DioEgizio** [comment]
wdym for cracked mode? We're not gonna allow you to use offline mode without an account, it's out of question

**Comment by svin24** [comment]
@DioEgizio I am not talking about piracy(although technically since it breaks microsoft's eula i guess i am)
I have logged into the client with my normal paid account. 
Can i set up a "cracked" profile that will let me play on "cracked" servers

**Comment by DioEgizio** [comment]
You can already join cracked servers on a paid account, and what you're talking about is offline mode/offline accounts, which already exist on PolyMC (you can add them only if you have a paid account)

**Comment by svin24** [comment]
Oh i see, testing it with some "cracked" servers it already seems to work. I guess this means PolyMC is worstcasescenario-proof
